### DISCUSSION TITLE
Question: Can we publish data generated with CARLA?

### DISCUSSION THREAD

**Discussion by Mariusmarten** [carla-simulator/carla#7081] [Q&A]
We are planning to publish a dataset of generated CARLA data. What do we need to take into consideration with regard to licensing? Anything pitfalls to look out for? The dataset will be freely available, and we will, of course, reference CARLA.

**Comment by SExpert12** [comment]
Hi,
Have you created a dataset?

**Reply by Mariusmarten** [reply]
Hi, yes exactly, we created a large multi-view dynamic dataset using Carla and plan on publishing it soon :)

**Comment by SExpert12** [comment]
Thanks for reply. Which version of CARLA you have used? Can you share information like which sensors and measurements also taken to collect the data?

**Reply by Mariusmarten** [reply]
This a subset of the data we plan to release: https://www.kaggle.com/datasets/mkaestingschaefer/seed4d, we used Carla 9.14

**Comment by SExpert12** [comment]
Great!!. Will you release or share code of the same?

**Reply by Mariusmarten** [reply]
We are still in the process, but I am hopeful that we can share the data and the code soon.
